OSSM-8740 Document full list of SMCP CR -> Istio CR fields mapping (#88602)
openshift/openshift-docs: modules/ossm-migrating-reference-unsupported-configurations.adoc

// Module included in the following assemblies:
//
// * service-mesh-docs-main/migrating/reference/ossm-migrating-reference.adoc

:_mod-docs-content-type: REFERENCE
[id="ossm-migrating-reference-unsupported-configurations_{context}"]
= Unsupported configurations in Service Mesh 3

> The following tables list {SMProduct} 2 `ServiceMeshControlPlane` configurations that are not supported in {SMProductName} 3. Remember that this does not necessarily mean the functionality has been removed. It can mean that the configurations can be managed separately in {SMProduct} 3, outside your `Istio` resource.

Reviewer: ```suggestion
The following tables list {SMProduct} 2 `ServiceMeshControlPlane` fields that are not supported in {SMProductName} 3. Remember that this does not necessarily mean the functionality has been removed. For example in case of addons, they are no longer managed by {SMProduct} 3 operator and must be installed as standalone applications. 
```

> == Add on Configurations

Reviewer 2: ```suggestion
== Add-on configurations
```
ISG asks to hyphenate add-on. Sentence case for headings.

> == Add on Configurations

Reviewer 2: Repeat for a new module: New headings should trigger unique anchor IDs. You could also consider further modularizing this given how many sections there are.

> Remember that add ons, such as {DTProductName}, {KialiProduct}, and others, are managed and configured separately in {SMProduct} 3. See "Observability and Service Mesh" for more information on how to configure them.

Reviewer 2: ```suggestion
Add-ons, such as {DTProductName}, {KialiProduct}, and others, are managed and configured separately in {SMProduct} 3. For more information, see "Observability and Service Mesh".
```

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentences

> [cols="1"]

Reviewer 2: Would table labels help in structuring all of this?

Author: Good thought. Thanks! That is a post-GA consideration.
